docs/contributing/release_process.md

```diff
@@ -41,38 +44,74 @@ unless this is impossible for security reasons or to prevent data loss.
 So the answer to "should I upgrade to the latest patch release?" will always
 be "yes."
 
+A feature release will usually stop receiving patch release updates when the next feature release comes out.
+
 **Long-term support release**
 
 Certain feature releases will be designated as long-term support (LTS)
 releases. These releases will get security and data loss fixes applied for
-a guaranteed period of time, typically six months.
+a guaranteed period of time. Typically, a Long Term Support release will happen
```

Comment:
Nitpicky but we use `Long-term support release` twice in the above text and your change has `Long Term Support release`.

Might be good to be consistent, whichever way we go.